src/Core/Checkout/Customer/SalesChannel/AbstractAccountService.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Checkout\Customer\SalesChannel;
+
+use Shopware\Core\Checkout\Customer\CustomerEntity;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+
+#[Package('checkout')]
+abstract class AbstractAccountService
+{
+    abstract public function getDecorated(): AbstractAccountService;
+
+    abstract public function setDefaultBillingAddress(string $addressId, SalesChannelContext $context, CustomerEntity $customer): void;
+
+    abstract public function setDefaultShippingAddress(string $addressId, SalesChannelContext $context, CustomerEntity $customer): void;
+
+    /**
+     * @deprecated tag:v6.7.0 - Method will be removed, use `AccountService::loginById` or `AccountService::loginByCredentials` instead
+     */
+    abstract public function login(string $email, SalesChannelContext $context, bool $includeGuest = false): string;
```

Comment:
the new abstract class should not have already deprecated stuff

Reply:
Removed the deprecated function in https://github.com/shopware/shopware/pull/3727/commits/65ddefacfe73f7c590d4acfed239fc8279cc16c4